analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -99,6 +101,13 @@ def get_argparser_ctor_args():
 hash will not be changed so easily for example on code indentation or when a
 checker is renamed.
 
+Exit status
```

Comment:
It would be good to add a similar section to the CodeChecker log command help. Maybe that can be added in a separate commit or in a separate pull request.

Reply:
According to my experiences the exit status of CodeChecker log is 1 if the last compiler invocation fails, 0 otherwise. We should change its behavior so it is 1 in case any build action fails. We should create a ticket for this.